UnityProject/Assets/Scripts/NPC/AI/Hostile/BlobAI.cs

```diff
@@ -79,7 +79,7 @@ private void StatusLoop()
 				}
 
 				searchWaitTime += MobController.UpdateTimeInterval;
-				if (searchWaitTime >= searchTickRate)
+				if (searchWaitTime >= searchTickRaten)
```

Comment:
"Raten"

Reply:
is because can't go through all the prefabs and update the values so just rename it work

Reply:
ah, rip.